planning/read_data.py

```diff
@@ -1,6 +1,6 @@
 import pandas as pd
 
-df = pd.read_csv('./google-sheet.csv')
+df = pd.read_csv('./planning/google-sheet.csv')
```

Comment:
Why the change to the file path, considering read_data.py and google-sheet.csv are in the same folder?